app/views/projects/show.html.erb

```diff
@@ -1,66 +1,14 @@
-        </th>
-      </tr>
-    </thead>
-
-    <tbody id="stories" data-url="<%= sort_project_path(@project.id) %>">
-      <% if @stories.present? %>
-        <% @stories.each do | story | %>
-          <tr class="project-table__row project-table__row--story" id="<%= dom_id(story)%>" >
-            <td class="project-table__cell">
-              <input type="checkbox" name="stories[]" value=<%= story.id %>>
-              <%= link_to story.title, [story.project, story] %>
-            </td>
-            <td class="project-table__cell"><%= story.estimate_for(current_user)&.best_case_points %></td>
-            <td class="project-table__cell"><%= story.estimate_for(current_user)&.worst_case_points %></td>
-            <td class="project-table__cell">
-              <%= link_to 'Clone', new_project_story_path(@project.id, story_id: story.id), class: "button green clone" %>
-              <%= link_to 'Edit', edit_project_story_path(@project.id, story), class: "button green edit" %>
-              <% if estimated(story) %>
-                <%= link_to 'Edit Estimate', edit_project_story_estimate_path(@project.id, story, @estimate_id), class: "button green edit-estimate", remote: true %>
-              <% else %>
-                <%= link_to 'Add Estimate', new_project_story_estimate_path(@project.id, story), class: "button green add-estimate", remote: true %>
-              <% end %>
-              <%= link_to 'Delete', project_story_path(@project.id, story, format: "json"), method: "delete",  data: { confirm: 'Are you sure?', story_id: story.id }, class: "button green delete", remote: true %>
-              <% if @siblings.any? %>
-                <div class="move-story-wrapper">
-                  <button class="button green">Move to</button>
-                  <div class="move-story-dropdown">
-                    <% @siblings.each do |to_project| %>
-                      <%= button_to to_project.title, project_story_move_path(@project, story, to_project: to_project), method: :put, data: {confirm: "Are you sure?"} %>
-                    <% end %>
-                  </div>
-                </div>
-              <% end %>
-            </td>
-          </tr>
-        <% end %>
-      <% else %>
-          <p> You don't have any stories yet. </p>
-      <% end %>
-    </tbody>
-
-    <tfoot>
-      <tr class="project-table__row project-table__row--footer">
-        <td class="project-table__cell">Total estimates</td>
-        <td class="project-table__cell"><%= @project.best_estimate_sum_per_user(current_user) %></td>
-        <td class="project-table__cell"><%= @project.worst_estimate_sum_per_user(current_user) %></td>
-        <td class="project-table__cell"></td>
-      </tr>
-    </tfoot>
-
-  </table>
+  <%= search_form_for @q, url: project_path(id: @project.id) do |f| %>
+    <div class="field">
+      <%= f.label :title_cont, "Filter by title" %>
```

Comment:
That's how ransack matches fields. [Here](https://github.com/activerecord-hackery/ransack/wiki/Basic-Searching) the explain what suffixes we can add. In this case it just means `title contains`

Reply:
I see.

I think it would be better to map something like `search_term` to `title_cont` in the backend.
I might be wrong, but it seems like we are putting some implementation details that don't belong in a view here.

I think a future developer might not understand what `_cont` stands for and making it more explicit could be useful.

Reply:
Just read Ariel's response, so we can ignore my reply above :-)